### PR TITLE
Add advanced layouts to Subsite Page content type

### PR DESCRIPTION
**Pull request by markconroy**

### CODE REVIEW

File: config/default/field.field.node.localgov_subsites_page.localgov_subsites_content.yml [hunk 1]
```diff
@@ -178,4 +180,7 @@ settings:
       topic_list_builder:
         weight: -64
         enabled: false
+      webform_response_item:
```
Comment (Polynya) [inline config/default/field.field.node.localgov_subsites_page.localgov_subsites_content.yml:183]: Where has this come from? I think it should be removed.

**Comment by Polynya** [comment]
Please change the PR to merge into develop instead of main.

File: config/default/field.field.node.localgov_subsites_page.localgov_subsites_content.yml (hunk 1 above)
Comment (markconroy) [reply config/default/field.field.node.localgov_subsites_page.localgov_subsites_content.yml]: Well spotted, thanks

**Comment by siliconmeadow** [comment]
Hi @markconroy - there's been a security update for composer that caught us on the hop on Friday. It's been merged into develop now, so you'll need to merge develop into this to get the checks to pass. Sorry about that. See #390

**Comment by markconroy** [comment]
Can this not "skip the queue" without going through the merge to develop merge to preprod promote to prod dance? 

I thought we agreed we can do that now? Treat this like a hotfix

**Comment by Polynya** [comment]
@markconroy the problem isn't really about merging but testing. Until we have environments per branch, we need to deploy to Preprod to test it. I'll discuss this with Will at the Essex LGD Dev meeting tomorrow.

**Review by Polynya — APPROVED** [review]

**Pull request merged**